guides/source/development_dependencies_install.md

```diff
@@ -123,8 +123,13 @@ $ sudo apt-get install sqlite3 libsqlite3-dev mysql-server libmysqlclient-dev po
 
 # Install Yarn
 # Use this command if you do not have Node.js installed
-$ curl --fail --silent --show-error --location https://deb.nodesource.com/setup_18.x | sudo -E bash -
+# ref: https://github.com/nodesource/distributions#installation-instructions
+$ sudo mkdir -p /etc/apt/keyrings
+$ curl --fail --silent --show-error --location https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+$ echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_18.x nodistro main" | sudo tee /etc/apt/sources.list.d/nodesource.list
```

Comment:
Lets updated to 20 while we are here, current lts- https://nodejs.org/en

Reply:
Thank you for your review.
I fixed it in d7396bfc20c3d20b5376fb2ece86c0b9694f76ca